packages/block-library/src/page-list/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * WordPress dependencies
+ */
+import { pages as icon } from '@wordpress/icons';
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit.js';
+
+const { name } = metadata;
+
+export { metadata, name };
+
+export const settings = {
+	title: _x( 'Page List', 'block title' ),
+	description: __( 'Display a list of all pages.' ),
+	icon,
+	example: {},
+	edit,
+};
```

Comment:
Do we need keywords? Maybe not, I can't think of any.

Reply:
'Menu' could be one, given it'll be useful for building menus. Also 'Navigation'?

Reply:
I can't think of any either 😅  We can always add them later if needed?